src/Universe/Behaviors/ActivationBehavior.cs

```diff
@@ -4,7 +4,6 @@
 //   subject to the Microsoft Public License.  All other rights reserved.
 // </copyright>
 // <summary>
-//   Created: June 2009 by Karak.
 //   Updated: November 2009 by bengecko - AbstractItemBehavior
```

Comment:
nit: Don't need the Updated - might be worth searching for `Updated: ` to see if you missed any others in the solution?